### PR TITLE
Added tyde menu

### PR DESCRIPTION
**Pull request by narendrasss**
Implements #149

### CODE REVIEW

File: templates/vue/src/components/Tyde.vue [hunk 1]
```diff
@@ -1,20 +1,97 @@
...
 </script>
 
-<style scoped>
-
+<style>
+:root {
+  --gray: #404040;
+  --light-gray: #787878;
+  --med-gray: #535353;
+  --light-blue: #1eade1;
```
Comment (wynnset) [inline templates/vue/src/components/Tyde.vue]: If these are global variables, we might want to prefix them with something like `tyde-` or `tapestry-` so it doesn't cause conflicts with potentially other things naming the variables the same thing.

**Comment by narendrasss** [comment]
Waiting on tyde #160

**Comment by narendrasss** [comment]
@wynnset I fixed the logs, but I need to update the way the menu handles lightboxes since it is now out of `tapestry.js`.

**Comment by wynnset** [comment]
Has this been tested and working with mp4 videos, h5p videos, and also other content types?

**Comment by wynnset** [comment]
Needs to be tested with H5P to make sure it's working. Otherwise good to go.

**Comment by wynnset** [comment]
Error: When I try to change the option "Allow skipping video if user has not watched at least once", the application throws console errors and fails to complete the operation.

**Pull request merged**